### PR TITLE
get the book ready for publishing

### PR DESCRIPTION
**Pull request by spring4175**
Get the book ready for publishing by updating the section on crates, removing the "Building a Bot" section, and simplifying the "Services" section. As Twilight is more targeted towards experienced Rust and bot developers, we don't need a tutorial for writing a bot.

### CODE REVIEW

File: src/chapter_1_crates/section_4_cache_inmemory.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Twilight includes an in-process-memory cache. It's responsible for processing
+events and caching things like guilds, channels, users, and voice states.
+
+## Installation
+
+Add the following to your Cargo.toml:
+
+```toml
+[dependencies]
+twilight-cache-inmemory { branch = "trunk", git = "https://github.com/twilight-rs/twilight" }
```
Comment (7596ff) [inline src/chapter_1_crates/section_4_cache_inmemory.md]:
```suggestion
twilight-cache-inmemory = { branch = "trunk", git = "https://github.com/twilight-rs/twilight" }
```

File: src/chapter_2_multi-serviced_approach.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+you have a small bot and just want to get it going in a monolithic application,
+then it's also a good choice. It's easy to split off parts of your application
+into other services as your application grows.
+
+## Gateway clusters
+
+One of the popular design choices when creating a multi-serviced application is
+to have a service that simply connects shards to the gateway and sends the
+events to a broker to be processed. As bots grow into hundreds or thousands of
+shards, multiple instances of the application can be created and clusters -
```
Comment (7596ff) [inline src/chapter_2_multi-serviced_approach.md:16]: hundreds of thousands of `shards` or `guilds`?

File: src/support.md [hunk 3]
```diff
@@ -10,19 +15,19 @@ The API documentation is [also available][api docs].
...
 
-Twilight for now follows the latest released stable.
+Twilight currently supports the latest stable Rust version.
 
 ### Breaking Changes
 
 Although Twilight aims to design APIs right the first time, that obviously won't
-happen. A lot of care is taken into designing them in an unopinionated way,
-leaving more opinionated concepts to userland libraries.
+happen. A lot of care is taken into designing clear and correct interfaces.
```
Comment (7596ff) [inline src/support.md]: `A lot of effort is spent designing clear and correct interfaces`, or `Care is taken while designing...`

**Review by 7596ff — CHANGES_REQUESTED** [review]

File: src/chapter_2_multi-serviced_approach.md (hunk 2 above)
Comment (spring4175) [reply src/chapter_2_multi-serviced_approach.md:16]: Shards is correct here

**Comment by spring4175** [comment]
Addressed comments

**Review by 7596ff — APPROVED** [review]

**Pull request merged**